website/docs/language/expressions/custom-conditions.mdx

```diff
@@ -29,9 +29,9 @@ For more information on when to use certain custom conditions, see [Choosing Bet
 
 ## Input Variable Validation
 
--> **Note:** Input variable validation is available in Terraform v0.13.0 and later.
+-> **Note:** Input variable validation is available in Terraform v0.13.0 and later. Before Terraform v1.9.0, validation rules can refer only to the variable being validated.
```

Comment:
```suggestion
-> **Note:** Input variable validation is available in Terraform v0.13.0 and later. Before Terraform v1.9.0, validation rules can refer only to the variable being validated, not other variables.
```

Reply:
This seems reasonable!

I kept this short both because we typically use these callouts mainly to nudge a reader that they might want to refer to the equivalent docs in an older version (now that the docs site supports that) and because older versions of Terraform will generate a suitable error message if someone tries to refer to anything else and so this is here mainly to answer the question "why did the example I found on the internet suggest trying something that returns an error?".

However, "not other variables" is a very reasonable addition that clarifies the point without adding too many more words. I'll update this before I merge.